OloEditor/src/Panels/VisualScriptEditorPanel.cpp

```
@@ -0,0 +1,1370 @@
+
+        // Below roughly a third scale, glyphs are unreadable mush; drawing them
+        // costs a lot and communicates nothing. The coloured boxes and wires
+        // still convey the graph's shape at that zoom.
+        if (zoom < 0.35f)
+        {
+            return;
+        }
+
+        //-- Title -----------------------------------------------------------------
+        const std::string& title = layout.m_Type != nullptr ? layout.m_Type->m_DisplayName : std::string("<unknown node>");
+        draw->AddText(ImVec2(topLeft.x + 8.0f * zoom, topLeft.y + 5.0f * zoom), IM_COL32(240, 240, 240, 255), title.c_str());
```

Cppcheck (2.21.0)

[error] Using pointer that is a temporary.

(danglingTemporaryLifetime)